content/en/blog/2024/otel-cicd-sig/index.md

```diff
@@ -0,0 +1,284 @@
+---
+title: OpenTelemetry Is Expanding Into CI/CD Observability
+linkTitle: OpenTelemetry Is Expanding Into CI/CD Observability
+date: 2024-12-01
```

Comment:
```suggestion
date: 2025-02-18
```

let's publish tomorrow (or Wednesday) since we have the ContribEx Survey today and OTTL context inference is aiming for the 20th